internal/ocm/client.go

```diff
@@ -45,9 +59,9 @@ func WithEndpoint(endpoint string) Option {
 	}
 }
 
-func WithClusterID(clusterID string) Option {
+func WithClusterExternalID(clusterExternalID string) Option {
 	return func(o *ClientOptions) {
-		o.ClusterID = clusterID
+		o.ClusterExternalID = clusterExternalID
 	}
 }
```

Comment:
Are we replacing `ClusterID` with `ClusterExternalID` or are we supporting both?
Because currently this PR still contains both variants.